src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/sweeper/JobSweeper.java

```diff
@@ -218,7 +219,7 @@ void sweep(ShardId shardId, String docId, Long version, BytesReference jobSource
             this.sweptJobs.put(shardId, jobVersionMap);
         }
         jobVersionMap.compute(docId, (id, currentVersion) -> {
-            JobDocVersion newJobVersion = new JobDocVersion(primaryTerm, seqNo, version);
+            JobDocVersion newJobVersion = new JobDocVersion(version.getPrimaryTerm(), version.getSeqNo(), version.getVersion());
```

Comment:
Why do you need to do this? Just use the given version from parameter for comparison?

Reply:
updated, fixed this silly code